CSVParser/ParsersOVerview.drawio.svg

```diff
@@ -0,0 +1,327 @@
+<svg host="65bd71144e" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="412px" height="461px" viewBox="-0.5 -0.5 412 461" content="&lt;mxfile&gt;&lt;diagram id=&quot;nucjY-XasSFz65NB0XN_&quot; name=&quot;Page-1&quot;&gt;7Vxdd5owGP41nrNduAMJH3pZtd0u1tNt7c7ayxSicArExVh1v34BAhjAftqASi9K8pI3CQ95njeEYA+Ow/VXiubeJXFx0AOau+7BSQ+AoWHy/7FhkxpszUoNM+q7qUkvDNf+PyyMmrAufRcvpIKMkID5c9nokCjCDpNsiFKykotNSSC3OkczXDFcOyioWv/4LvNS6wDYhf0b9mde1rJuDdMzIcoKiytZeMglqy0TPO/BMSWEpalwPcZBjF2GS+p3seNs3jGKI/YiBwH8gm2yi8Muv1aRjUjEDyOPhQHP6TyJ1z675Wntiylyd3FOpCfr7cxGZNIm4np39lKYFmRJHVEK5HDwYYRJiBnd8CIUB4j5j3JVSNzQWV4ud/1BfN4I0MTYg4ZAXgw9S5NrYIjOMBNOBXA8sdWLwpTAWQ8tSCt+RMFSdLYHrIBf78j1HyXIrb/L+H6PHBIQ2oNn8VXO7j9Bfm9473hDmqEXSetznI5R1e6R8zCjZBm5/ZIvMM3co5Qu3KckYv0pCv1gk3ryrqBwnpyE0ODHEEcBqVjz2qrlSYSc1zg4/Ib7mPJTEV7t9uP1ksUc8aGR9T3wI9z3BMnivuuD+bqAkqdm4vgk5CcAwfjX94sMBj5mEyRkcMrWkiAkAwy7gswrz2f4OukInKy4vMv6IAY9pgyvn2Z7ldrCwdANiaJDQdFVIbUZa70tlYXabi2QaPwUZw0FcshhoJstpzh7l9UXZwq3JPc+GYUiPia6JgLVO5X1xWiap4CmrimCE+qnAKepCs0uPLcgNl1dXp61NAxBU54pQpVxaHh0VDeqVLcUUd3oqN481fmY7eOFg+acwu0kvA5kwtdNPO0awpt7IHwmGkdEeLM5wpsd4ZsnfLvJDkrrQFBTGN2PbyJvVcmu7CkTtBjNBaPkAY9TheBdg+Pkb58424pwtjpRbV5Upz4O2iqpulWaPwGFkgpPSATsqggoW4SyX60Cz5A6Z3RO53ouC+oK9iSslBiYa04dAXMWyeR7gnkJ5cqPMCjEH8G9+FrrRsdz4+Z9bLVltubsVcBW3W4xW9/GyUGVk0NFlBx0gbn5wEyxQ2hbIzPUZa4rXMkcVgbn1OcwtA0RqFD82vxWbN9TlQzmJnQxa/sk5yoeRm5c9ZHMVkBzCqYPWszXN7ISVFkJVLESdNOV5qcr3b6o0YHtioLl5zVbnQSCI5TAJreetHmxan9wQmUhpdvL0wI97ULKaPLz99XN+UGFFX0ghxVD4TJg9krpmHSwZpuTqrACjm9VtQ5OQ1VU6baNtUBSu6jSQQBHN+e3N5Ozm51bhQ8p4Ga705rYVqxil+Eblf69q8LCtfS5n1Ha5pXv8dz/93569VVHFy86sVQPwfjXQQmi0eDCFjzC7/1q1vaVLWx1a/st4H8ngSNwgCsxoKSDcPBhOsizxU9MpFOd4nc64Pl/&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
This filename has an uppercase "V" on OVerview, so the link doesn't work in github.

Reply:
Fixed